support/k8s/k8s-workload-registrar/mode-crd/controllers/utils.go

```diff
@@ -131,8 +132,14 @@ func equalStringSlice(x, y []string) bool {
 
 func containsString(slice []string, s string) bool {
```

Comment:
The `containsString(...)` function is called from a few difference places, only some of which are related to checking for namespace matches. It might be worthwhile to create a entirely new function that will check for an equal or regex match, and then call this new function from the places where the namespace is being compared.

Reply:
yes, good catch. Lets create a new function